src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitManager.java

```diff
@@ -21,15 +21,17 @@
   private Coordinator coordinator;
   private RecoveryHandler recovery;
   private CommitHandler commit;
-  private String namespace;
-  private String tableName;
+  private Optional<String> namespace;
+  private Optional<String> tableName;
 
   @Inject
   public ConsensusCommitManager(DistributedStorage storage) {
     this.storage = storage;
     this.coordinator = new Coordinator(storage);
     this.recovery = new RecoveryHandler(storage, coordinator);
     this.commit = new CommitHandler(storage, coordinator, recovery);
+    this.namespace = storage.getNamespace();
```

Comment:
It now inherits the default namespace and the table name from `DistributedStorage`.